etc/visual-studio/libopenthread-ncp-uart.vcxproj

```diff
@@ -73,4 +76,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
yea, confusing:

I learned a new word here.   https://octicons.github.com/icon/no-newline/

Take a look at this earlier commit etc/visual-studio/libopenthread-ncp-uart.vcxproj

Question: What happened at line 1, and line 73/75? 

Answer: Line 1 - contained a byte order mark (U+FFEF), this is the commit that the byte order mark was removed.

Answer: Line 73/75 - *DID* not have newline, it was added during this commit.

Note: My commit does not *NOT* have the byte order mark, (this is something that @darconeous  noticed... in my commits, I fixed these.

Now fast forward to the current problem,

Compare my two GIT HUB commits, the only change is the octicon .. moved from one line to the other.
My change, adding/removing the terminal newline.

Surely I cannot delete what is on Github...  But Github says, in one case - the old file is missing a terminal newline, and in the other, commit the old file has a terminal newline.

I wonder if this is a github bug in the rendering/display code for bitbucket.

https://github.com/openthread/openthread/commit/9301cf12d7b41ec5970f25f7e2529f8a9f894949

https://github.com/openthread/openthread/commit/9301cf12d7b41ec5970f25f7e2529f8a9f894949

I can make this any way you want, - my experience is this:

a)  Visual studio would like - but does not require a U+FFEF byte order mark.

b)  Visual studio does not require, and seems to ignore a terminal newline.

Visual studio seems to follow Postal's Law.